Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py

```diff
@@ -1119,13 +1108,20 @@ def main():
 
     params = demisto.params()
 
+    # init credentials
     user_name = params.get('username')
     api_key = params.get('apikey')
     server_url = params.get('url', '').strip('/')
-
     CREDENTIALS['username'] = user_name
     CREDENTIALS['api_key'] = api_key
 
+    # init threshold from params
+    THRESHOLDS_FROM_PARAM['url'] = arg_to_number(params.get('url_threshold'))
+    THRESHOLDS_FROM_PARAM['ip'] = arg_to_number(params.get('ip_threshold'))
+    THRESHOLDS_FROM_PARAM['file'] = arg_to_number(params.get('file_threshold'))
+    THRESHOLDS_FROM_PARAM['domain'] = arg_to_number(params.get('domain_threshold'))
+    THRESHOLDS_FROM_PARAM['threatstream-email-reputation'] = arg_to_number(params.get('email_threshold'))
```

Comment:
```suggestion
    THRESHOLDS_FROM_PARAM = {
         'url' = arg_to_number(params.get('url_threshold')),
         'ip' = arg_to_number(params.get('ip_threshold')),
         'file' = arg_to_number(params.get('file_threshold')),
         'domain' = arg_to_number(params.get('domain_threshold')),
         'threatstream-email-reputation' = arg_to_number(params.get('email_threshold')),
    }
```

Reply:
Are you sure?
it a global param and to init it as suggested we need to declare it 
`global THRESHOLDS_FROM_PARAM `
from my point of view the current implementation is more intuitive and clean

Reply:
It is up to you to decide which way to go here.
I have full faith in your decision.